lib/pagseguro/refund/response.rb

```diff
@@ -1,22 +1,33 @@
 module PagSeguro
   class Refund
     class Response
-      extend Forwardable
-
-      def_delegators :response, :success?
-      attr_reader :response
-
-      def initialize(response)
+      def initialize(response, refund)
         @response = response
+        @refund = refund
       end
 
-      def errors
-        @errors ||= Errors.new(response)
+      def serialize
+        if response
```

Comment:
@dmitryrck here it's better to have `if success?`. response can exist, but with errors.